### PR TITLE
[ZEPPELIN-4555]. paragraph title is disappear after I run the paragraph

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?
This PR is to fix the title disappear issue.  The main change is in method `mergeConfig` where I would always insert the config in `interpreter-setting.json`

### What type of PR is it?
[Bug Fix ]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-4555

### How should this be tested?
* CI pass

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by zjffdu** [comment]
@liuxunorg Could you help review it ? Thanks

File: zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/NotebookTest.java [hunk 1]
```diff
@@ -1123,29 +1123,35 @@ public void testInterpreterSettingConfig() {
...
-    note.runAll(AuthenticationInfo.ANONYMOUS, false);
+    p1.execute(true);
 
-    // wait until first paragraph finishes and second paragraph starts
-    while (p1.getStatus() != Status.FINISHED) Thread.yield();
+    // mock1 has no config setting in interpreter-setting.json, so keep the previous config
+    LOGGER.info("changed intp p1.getConfig() =  " + p1.getConfig());
+    assertEquals(config.get(InterpreterSetting.PARAGRAPH_CONFIG_RUNONSELECTIONCHANGE), false);
+    assertEquals(config.get(InterpreterSetting.PARAGRAPH_CONFIG_TITLE), true);
+    assertEquals(config.get(InterpreterSetting.PARAGRAPH_CONFIG_CHECK_EMTPY), false);
```
Comment (liuxunorg) [inline zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/NotebookTest.java]: `config_test` interpreter's setting checkEmpty=false
```
      "config": {
        "fontSize": 9,
        "colWidth": 12,
        "runOnSelectionChange": false,
        "title": true,
        "checkEmpty": false
      },
```

Change paragraph test to `p1.setText("%mock1 sleep 1000");`, 
But `mock1` interpreter's unsetting checkEmpty.
`InterpreterSetting.PARAGRAPH_CONFIG_CHECK_EMTPY` Should the default value be equal to `true`??

Comment (zjffdu) [reply zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/NotebookTest.java]: @liuxunorg The problem here we could not override the config with the config in interpreter-setting. Basically config could from 3 sources when user change interpreter in one paragraph:
* the old interpreter's default config
* the new interpreter's default config
* config made by user manually

Actually we could not decide where the config come from, that's why I will keep the previous config which I think is the best approach to handle this https://github.com/bbonnin/zeppelin-mongodb-interpreter

**Comment by zjffdu** [comment]
Thanks @liuxunorg

**Pull request closed without merging**